docs/b2b-edition/specs/api-v3/order/order.yaml

```diff
@@ -1,20 +1,121 @@
+
+    Note that this resource **does** not allow you to create new orders, update an order’s basic information, or add attributes like shipments and order-level fees. Use the BigCommerce [Orders V2 API](/docs/rest-management/orders) to handle these tasks in your integration.
+
+    When using B2B Orders endpoints to retrieve or update specific orders, use the **BigCommerce Order ID** as the unique identifier. The **B2B Edition Order ID**, presented as the `id` value of certain B2B Orders responses, is not a supported identifier for the `bcOrderId` field.
+
+    The `statusCode` field in some endpoint responses is the unique identifier assigned to each BigCommerce order status. To learn more about order statuses and their corresponding codes, see [Order Statuses](/docs/rest-management/orders/order-status).
+
+    ## Legacy Extra Fields
+
+    The response body of certain endpoints, such as [Get All Orders](#get-all-orders) and [Create an Order](#create-an-order), include the following legacy Company extra fields used by older stores:
+
+    - `extraInt1`
+    - `extraInt2`
+    - `extraInt3`
+    - `extraInt4`
+    - `extraInt5`
+    - `extraStr1`
+    - `extraStr2`
+    - `extraStr3`
+    - `extraStr4`
+    - `extraStr5`
+    - `extraText`
+
+    These fields return `null` values in newer stores. We recommend using the [Get Company Extra Fields](/b2b-edition/apis/rest-storefront/company/companies#get-company-extra-fields) endpoint to retrieve Company extra field information.
+
+    ## USD Currency Conversion
+
+    The `usdIncTax` field converts the order's total price from the currency it was placed into USD, even if you do not offer USD as a transactional currency. The following endpoints include the field in their response body:
+
+    - [Get All Orders](#get-all-orders)
+    - [Create an Order](#create-an-order)
+    - [Get an Order](#get-an-order)
+    - [Update an Order](#update-an-order)
   contact: {}
 servers:
   - url: 'https://api-b2b.bigcommerce.com/api/v3/io'
 tags:
-  - name: Order
-    description: Customer's Orders management.
+  - name: Orders
+    description: Managing orders placed by buyers on stores using B2B Edition.
 paths:
   /orders:
     get:
-      summary: Get Orders
+      summary: Get All Orders
       tags:
-        - Order
+        - Orders
+      operationId: get-orders
+      security:
+        - authToken: []
+      description: |-
+        Retrieves limited information for all orders, including B2B-specific information like Company IDs and order extra fields. The response includes orders placed by B2C customers and guest shoppers.Use [Get Order Products](#get-order-products) to retrieve an order's product details, or use the BigCommerce [Orders V2 API](/docs/rest-management/orders) to retrieve all order details except those specific to B2B Edition, like Company ID and extra fields.
```

Comment:
There's a space missing after the period after "placed by B2C customers and guest shoppers."